driver/controller.go

```diff
@@ -955,7 +955,11 @@ func extractStorage(capRange *csi.CapacityRange) (int64, error) {
 	}
 
 	if requiredSet && !limitSet && requiredBytes < minimumVolumeSizeInBytes {
-		return 0, fmt.Errorf("required (%v) can not be less than minimum supported volume size (%v)", formatBytes(requiredBytes), formatBytes(minimumVolumeSizeInBytes))
+		d.log.WithFields(logrus.Fields{
+			"requiredBytes":            formatBytes(requiredBytes),
```

Comment:
Would you mind using camel case here and on the field below in order to be consistent with the existing style?